src/frontend/sider/Fagsak/Behandling/Sider/Vilkårsvurdering/GeneriskVilkår/GeneriskVilkår.tsx

```diff
@@ -119,6 +124,13 @@ const GeneriskVilkår: React.FC<IProps> = ({
                 hideLegend
             >
                 <Heading size="medium" level="3">
+                    {skalViseManueltLagtTilISøknadLightbulb() && (
```

Comment:
Ettersom variabelnavnet er såpass beskrivende kan den godt være inline for min del
```suggestion
                    {person.erManueltLagtTilISøknad && (
```